src/mattack_actors.cpp

```diff
@@ -411,7 +411,7 @@ void gun_actor::shoot( monster &z, Creature &target, const std::string &mode ) c
     }
 
     tmp.weapon = gun;
-    tmp.worn.push_back( item( "fake_UPS", calendar::turn, 1000 ) );
+    tmp.i_add( item( "UPS", calendar::turn, 1000 ) );
```

Comment:
Crap, I merged too soon before I tested this. "UPS" is not an item, "UPS_off" or something like that is the item we want.

Can you fix it in #17020? That PR depends on UPS spawning right to test solar turrets.